src/shadowbox/server/server_access_key.ts

```diff
@@ -212,8 +220,11 @@ export class ServerAccessKeyRepository implements AccessKeyRepository {
     for (const accessKey of this.accessKeys) {
       const usageBytes = bytesTransferredById[accessKey.id] || 0;
       const wasOverDataLimit = accessKey.isOverDataLimit;
-      accessKey.isOverDataLimit =
-          this.accessKeyDataLimit ? usageBytes > this.accessKeyDataLimit.bytes : false;
+      let limitBytes = (accessKey.dataLimit || this._defaultDataLimit)?.bytes;
```

Comment:
This seems to be the recommended operator: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

```suggestion
      let limitBytes = (accessKey.dataLimit ?? this._defaultDataLimit)?.bytes;
```